src/DataTable/DataTable.scss

```diff
@@ -0,0 +1,109 @@
+
+$table-border: 1px solid $gray-200!default;
```

Comment:
Should we rename these variables to `$data-table-...` and the class names to `.png_data-table-...`?

Reply:
Yes, yes we should.